package.json

```diff
@@ -24,8 +24,9 @@
   },
   "homepage": "https://github.com/RocketCommunicationsInc/ag-grid-theme#readme",
   "devDependencies": {
-    "@astrouxds/design-tokens": "~2.0.0-beta.5",
     "@astrouxds/astro-web-components": "^6.0.5",
```

Comment:
Does this not need updated to astrouxds/astro-web-components@latest?

Reply:
nope, removed web comps and the old token repos entirely